content/en/docs/concepts/instrumentation/libraries.md

```diff
@@ -67,6 +101,26 @@ still useful in others.
 The rest of this document gives guidance on what and how to instrument if you
 decide to do it.
 
+## Personally Identifiable Information (PII)
+
+In some semantic conventions, the data collected as a span attribute could
+contain PII (Personally Identifiable Information). As a general guideline, do
+**not** collect this data by default. A few other options for handling this data
+are:
+
+- Provide opt-in features to collect potentially PII for users who need it
+- Collect it but obfuscate / redact the PII, if
+  you’re confident about being able to implement the obfuscation / redaction
+  effectively, non-reversible and with little overhead
+
+PII is data like email addresses, passwords, usernames, custom data in database
+queries, etc. For example, this is a Elasticsearch query with PII redacted that
+would be a span's `db.statement` attribute:
+
+```json
+{ "query": { "match": { "username": "REDACTED" } } }
+```
+
```

Comment:
```suggestion
```

@estolfo it looks like the PII discussion will take a little bit longer, this should not stop us from merging your other changes into the page. I argue that we should move this into a separate (follow-up) PR for discussion and merge the other work already. wdyt? 

Let me add, that I highly appreciate your efforts to dig into PII and to provide some documentation around it. This is a _really_ tough topic, but one we should definitely address in our documentation. Thank you!!

Reply:
Hi @svrnm Thanks for your review and suggestions. No problem to come back to the PII discussion in a separate PR.